xep-0156.xml

```diff
@@ -168,6 +174,7 @@ _xmppconnect IN TXT "_xmpp-client-websocket=wss://web.example.com:443/ws"
       <li>HTTP queries for host-meta information MUST be used only as a fallback after the methods specified in <cite>RFC 6120</cite> have been exhausted.</li>
       <li>A domain SHOULD NOT present information in host-meta link records that is available via the DNS SRV records defined in <cite>RFC 6120</cite>.</li>
       <li>The order of XMPP related link entries in the host-meta file SHOULD NOT be interpreted as significant by the presenting domain or the receiving entity.</li>
+      <li>Support for the XML version of the host-meta file is REQUIRED as specified by <cite>RFC 6120</cite> &sect;3 while the JSON version (and other representations) are OPTIONAL.</li>
```

Comment:
Minor but it sounds like it is the XEP that makes alternative representations optional.

I'd go with something like
```
As specified in RFC 6120, support for the XML/XRD encoding of the host-meta resource is REQUIRED while alternative representations such as JSON are OPTIONAL.
```